client/services/code_preprocessors/PythonCodePreprocessorService.js

```diff
@@ -165,9 +165,13 @@ tie.factory('PythonCodePreprocessorService', [
         VARNAME_CORRECTNESS_TEST_RESULTS + ' = []',
       ].join('\n');
       for (var i = 0; i < correctnessTests.length; i++) {
+        var testInputCode = (
+          inputFunctionName ? inputFunctionName + '(' + VARNAME_TEST_INPUTS + 
```

Comment:
Great suggestion. But I think it's not that easy to wrap it into a function, considering the argument could contain (, [, another function name and index, so I'll just go with the first one.